services/connected_account.yaml

```diff
@@ -16,29 +20,48 @@ Metadata:
       - Label:
           default: CloudZero Callback Information
         Parameters:
+          - ReactorId
           - ReactorSQSQueueUrl
     ParameterLabels:
+      AccountName:
+        default: |
+          Optional Name you can set for your AWS Account; you can use this as a descriptive alias for AccountId
       ExternalId:
         default: |
           Unique ExternalId for your Organization; Generated by CloudZero; Used for Cross Account Roles
       ReactorAccountId:
         default: |
           CloudZero AWS AccountID; Used for Cross Account Roles
+      ReactorId:
+        default: |
+          CloudZero Reactor ID; Used to Uniquely ID the CloudZero System Ingesting your Data
       ReactorSQSQueueUrl:
         default: |
           CloudZero Reactor SQS Queue receives callbacks from this template
 
 Parameters:
+  AccountName:
+    Type: String
+    Default: ''
+    Description: |
+      Optional Name you can set for your AWS Account; you can use this as a
+      descriptive alias for AccountId. It's useful to set this here when you
+      are automating provisioning many  accounts with this template.
```

Comment:
the editor in me won't let me unsee the extra whitespace in `many  accounts`

Reply:
LOL. Fixed.